python-package/lightgbm/callback.py

```diff
@@ -18,15 +18,15 @@ def _lt_delta(curr_score: float, best_score: float, delta: float) -> bool:
 class EarlyStopException(Exception):
     """Exception of early stopping."""
 
-    def __init__(self, best_iteration: int, best_score: float) -> None:
+    def __init__(self, best_iteration: int, best_score: List[Tuple[str, str, float, bool]]) -> None:
```

Comment:
It seems that this list can contain 5-element tuples.
https://github.com/microsoft/LightGBM/blob/67b4205c8043326553e294fa2c01ad1189784631/python-package/lightgbm/callback.py#L280
https://github.com/microsoft/LightGBM/blob/67b4205c8043326553e294fa2c01ad1189784631/python-package/lightgbm/callback.py#L275
https://github.com/microsoft/LightGBM/blob/67b4205c8043326553e294fa2c01ad1189784631/python-package/lightgbm/callback.py#L51

Reply:
oooo thank you! I checked the format of this with a debugger, but forgot you could ask to show standard deviation.

I'll update this hint.